ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -791,6 +791,56 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Security Auditing Is Enabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    macOS's audit facility, auditd, receives notifications from the kernel when certain system calls, such as open, fork, and exit, are made. These notifications are captured and written to an audit log.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy the following script which will enable security auditing:
+    /usr/bin/sudo /bin/launchctl load -w /System/Library/LaunchDaemons/com.apple.auditd.plist
+  query: |
+    SELECT 1 WHERE 
+      -- For all files in /var/audit UID should be owned by root
+      NOT EXISTS ( SELECT 1 FROM file WHERE path LIKE '/var/audit/%' AND uid !=0 )
+      AND
+      -- For all files in /var/audit GID should be owned by root
+      NOT EXISTS ( SELECT 1 FROM file WHERE path LIKE '/var/audit/%' AND gid !=0 )
+      AND
+      -- For all files in /var/audit MODE should be 0440 ("-r--r-----")
+      NOT EXISTS ( SELECT 1 FROM file WHERE path LIKE '/var/audit/%' AND mode != "0440" )
+      AND 
+      NOT EXISTS ( select 1 from file where path LIKE ( 
```

Comment:
Same with these three `SELECT`s, can be one similar to https://github.com/fleetdm/fleet/pull/9592/files#r1094666142.